main/swadge2024.h

```diff
@@ -354,6 +355,17 @@ typedef struct swadgeMode
      */
     fnDacCallback_t fnDacCb;
 
+    /**
+     * @brief This function is called to fill in a SwadgePass packet with mode-specific data. The Swadge mode should
+     * only fill in it's relevant data and not touch other mode's data.
+     *
+     * @warning This function will be called when the mode is not initialized or running, so it MUST NOT rely on memory
+     * allocated or data loaded in the mode's initializer.
+     *
+     * @param packet The packet to fill in
+     */
+    void (*fnAddToSwadgePassPacket)(struct swadgePassPacket* packet);
```

Comment:
For what it's worth, if you rebase, unrelated like this won't show up in the review.